src/linter.js

```diff
@@ -235,7 +235,7 @@ export default class Linter {
               var manifestParser = new ManifestJSONParser(
                 json,
                 this.collector,
-                {selfHosted: this.config.selfHosted},
+                {selfHosted: this.config.selfHosted, files:this.io},
```

Comment:
Oh and very minor nitpick but we've been doing `files: this.io` (note the space after colon).

We should have stylelint rules for it but we seem to lack them. I've filed #1069 for that.